ddtrace/contrib/pylons/middleware.py

```diff
@@ -57,7 +58,14 @@ def __call__(self, environ, start_response):
             # set analytics sample rate with global config enabled
             span.set_tag(ANALYTICS_SAMPLE_RATE_KEY, ddconfig.pylons.get_analytics_sample_rate(use_global_config=True))
 
-            trace_utils.set_http_meta(span, ddconfig.pylons, request_headers=request.headers)
```

Comment:
So is this a first step towards an event-based approach to tracing? 🙂

Reply:
yes!!!